packages/database/prisma/core-schema.prisma

```diff
@@ -17,17 +17,25 @@ enum AssignmentStatus {
   OUTSTANDING
 }
 
+type EncryptionKeyPair {
+  publicKey  Bytes
+  privateKey Bytes
+}
+
 model AssignmentModel {
-  createdAt        DateTime               @default(now()) @db.Date
-  updatedAt        DateTime               @updatedAt @db.Date
-  id               String                 @id @map("_id")
-  completedAt      DateTime?
-  expiresAt        DateTime
-  instrumentRecord InstrumentRecordModel?
-  status           AssignmentStatus
-  subject          SubjectModel           @relation(fields: [subjectId], references: [id])
-  subjectId        String
-  url              String
+  createdAt         DateTime               @default(now()) @db.Date
+  updatedAt         DateTime               @updatedAt @db.Date
+  id                String                 @id @map("_id")
+  completedAt       DateTime?
+  expiresAt         DateTime
```

Comment:
Optional expiry?

Reply:
I added a default expiry of one year. Much easier to do on client side than change the internal workings everywhere for this. If you think this should be longer, let me know and I can change in next PR.